src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
My point is that the `footer` element can be used for other footers than the overall page footer. You can have a `footer` in another part of the page as a semantic element within another block. I'm not suggesting we don't have a standard footer, but that we just put a class on this footer to differentiate it (eg `<footer class="mofo-footer">`).